src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -138,7 +130,7 @@ public void completed(AbstractBuild r) {
 
     String getChanges(AbstractBuild r) {
         if (!r.hasChangeSetComputed()) {
-            logger.info("No change set computed...");
+            logger.log(Level.FINER, "No change set computed...");
```

Comment:
All logger calls have been converted unless they needed to use .log()'s Throwable parameter